state-sync/src/coordinator.rs

```diff
@@ -1331,11 +1304,6 @@ impl<T: ExecutorProxyTrait> StateSyncCoordinator<T> {
             .version()
             .unwrap_or_else(|| known_version.wrapping_add(1));
         counters::set_version(counters::VersionType::Target, target_version);
-        let highest_version = self
```

Comment:
yeah, we should figure out this since you're not really getting rid of highest, you're getting rid of target

Reply:
I have another PR ready to fix this (as part of our effort to always return the highest ledger info regardless), so I've let it to that PR to fix this. Will send it out once this lands (as it depends on it) 😄

Reply:
As discussed offline, we'll just use the single highest_known_ledger_info or target.